src/openrct2-ui/windows/About.cpp

```diff
@@ -47,19 +47,19 @@ enum WINDOW_ABOUT_WIDGET_IDX {
 
 #define WIDGETS_MAIN \
     WINDOW_SHIM(WINDOW_TITLE, WW, WH), \
-    { WWT_IMGBTN,           1,  0,              WW - 1,         TABHEIGHT,  WH - 1,     0xFFFFFFFF,                             STR_NONE },             /* page background */ \
-    { WWT_TAB,              1,  3,              93,             17,         TABHEIGHT,  IMAGE_TYPE_REMAP | SPR_TAB_LARGE,       STR_NONE },             /* about OpenRCT2 button */ \
-    { WWT_TAB,              1,  94,             184,            17,         TABHEIGHT,  IMAGE_TYPE_REMAP | SPR_TAB_LARGE,       STR_NONE }              /* about RCT2 button */
+    MakeWidget     ({ 0, TABHEIGHT}, {WW, WH - TABHEIGHT}, WWT_IMGBTN, 1               ), /* page background */       \
+    MakeRemapWidget({ 3,        17}, {91,             34}, WWT_TAB,    1, SPR_TAB_LARGE), /* about OpenRCT2 button */ \
+    MakeRemapWidget({94,        17}, {91,             34}, WWT_TAB,    1, SPR_TAB_LARGE)  /* about RCT2 button */
```

Comment:
Stopped using `TABHEIGHT`

Reply:
`TABHEIGHT` is set to 50; I think it's a bit of a misnomer given that the actual height appears to be 34. I don't see the added value of using `TABHEIGHT - 16` here instead, if I'm honest.